### DISCUSSION TITLE
How to get Route 53 to work? Just getting "Host is not set"

### DISCUSSION THREAD

**Discussion by pullasuti** [qdm12/ddns-updater#757] [Q&A] [answered]
Thought I'd set ddns-updater running on a Proxmox container running Alpine. I downloaded the prebuilt 2.7 release. Everything seems fine, however when testing with two different domains and Route 53, I just keep getting "ERROR validating provider specific settings: host is not set". 

I have no idea where the error is coming from, it doesn't matter if I use correct or incorrect keys or domains. User permissions have also been set, I've previously been using an old ddns script on a now defunct Raspberry Pi to update the domains. 

Any ideas? I'm probably just overlooking something.

**Comment by qdm12** [comment, marked as answer]
What's your json configuration? You might be missing the `"host":"@"` key (see https://github.com/qdm12/ddns-updater/blob/v2.7.0/docs/route53.md#compulsory-parameters).
You might be reading the 'latest' readme, which doesn't correspond to the released binaries. See the versioned readmes section: https://github.com/qdm12/ddns-updater?tab=readme-ov-file#versioned-documentation

**Comment by pullasuti** [comment]
Yep, that was it! I tried going through all the docs for different ddns services to find a host key, but didn't find one. 

Was following this for Route53: https://github.com/qdm12/ddns-updater/blob/master/docs/route53.md which currently seems to miss the mentioned line. 

But all is good, thank you for a fast reply

**Reply by qdm12** [reply]
> Was following this for Route53: https://github.com/qdm12/ddns-updater/blob/master/docs/route53.md which currently seems to miss the mentioned line.

Indeed, that's because the last program (master branch readme) only uses the domain key and extracts the host from it (with retro-compatibility). And so check out the v2.7.0 readme and docs directory 😉 Not super obvious I agree, but we shall get over it eventually once a new release comes out.
